lib/ansible/plugins/inventory/gcp_compute.py

```diff
@@ -335,6 +338,12 @@ def parse(self, inventory, loader, path, cache=True):
         config_data = {}
         config_data = self._read_config_data(path)
 
+        # this may optionally be read from environment variable
+        if 'service_account_file' not in config_data:
```

Comment:
this conditional is not needed, just use get_option since if read_config_data already consumes the options in the config file